content/docs/ref/pr.md

```diff
@@ -32,27 +34,7 @@ Any [generic option](/doc/ref) in addition to:
 cml pr "."
 ```
 
-### Automatically merge GitHub pull requests
-
-```yaml
-on: pull_request
-jobs:
-  cml:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: iterative/setup-cml@v1
-      - name: Generate data
-        run: echo "Hello World" > output.txt
-      - name: Create and merge PR
-        run: |
-          cml ci
-          gh pr merge --rebase $(cml pr "output.txt")
```

Comment:
a few problems

- do we need a whole complete working YAML file? Seems excessive
  - `gh pr merge` & other GH-specifics not great - better to be CI agnostic
- `cml ci` is technically redundant in this case since `cml pr` already implicitly calls it. Maybe a better example which *really* needs `cml ci`?

Reply:
Note that this pull request deletes the section this comment is about. I assume you want it back, following my original https://github.com/iterative/cml.dev/pull/206#issue-1171610965 hesitation?

Reply:
Partially bring back in an improved form :)

Reply:
Please take another look.

Examples of `cml ci` should probably be in another section, not in the `cml pr` one. 🤔